docs/user-guide/advanced/replicate_evaluations.md

```diff
@@ -140,6 +139,16 @@ IN_FEATURES=1024 \
 eva predict_fit --config configs/vision/pathology/offline/<task>.yaml
 ```
 
+### Prov-GigaPath - DINOv2 ViT-G14
+To evaluate the `Prov-Gigapath` model available on [HuggingFace](https://huggingface.co/prov-gigapath/prov-gigapath):
```

Comment:
change to: `To evaluate the [Prov-Gigapath](https://github.com/prov-gigapath/prov-gigapath) model, available on [HuggingFace](https://huggingface.co/prov-gigapath/prov-gigapath), run:`

Also is there a paper we can reference?

Reply:
done